src/app/page.tsx

```diff
@@ -37,6 +45,13 @@ const Index = () => {
           value={apiKey}
           onChange={(e) => setApiKey(e.target.value)}
         />
+        <Input
```

Comment:
@klb3713 I would move this above API Key, since it will have a default value and user's would need to enter API Key.